src/main/java/frc/robot/controls/TestController.java

```diff
@@ -16,24 +24,51 @@ public class TestController {
   private GameController test = new GameController(OIConstants.kTestJoy);
   private FourBarArm m_arm;
   private Intake m_intake;
+  private Elevator m_elevator;
   
-  public TestController(FourBarArm arm, Intake intake){
+  public TestController(FourBarArm arm, Intake intake, Elevator elevator){
     m_arm = arm;
     m_intake = intake;
+    m_elevator = elevator;
   }
   
   public void configureControls() {
     
-    // elevator controls
-    test.get(Button.Y).onTrue(new ExtendToPosition(m_arm, ArmConstants.kTopPosition));
-    test.get(Button.X).onTrue(new ExtendToPosition(m_arm, ArmConstants.kMiddlePosition));
-    test.get(Button.A).onTrue(new ExtendToPosition(m_arm, ArmConstants.kIntakePosition));
-    test.get(Button.B).onTrue(new ExtendToPosition(m_arm, ArmConstants.kShelfPosition));
+    //arm controls
+      test.get(Button.Y).onTrue(new ExtendToPosition(m_arm, ArmConstants.kTopPosition));
+      test.get(Button.X).onTrue(new ExtendToPosition(m_arm, ArmConstants.kMiddlePosition));
+      test.get(Button.A).onTrue(new ExtendToPosition(m_arm, ArmConstants.kIntakePosition));
+      test.get(Button.B).onTrue(new ExtendToPosition(m_arm, ArmConstants.kShelfPosition));
     
     // intake controls
-    test.get(DPad.DOWN).onTrue(new InstantCommand(() -> m_intake.intake(IntakeConstants.kIntakeSpeed), m_intake));
-    test.get(DPad.UP).onTrue(new InstantCommand(() -> m_intake.intake(IntakeConstants.kOuttakeSpeed),m_intake));
-    test.get(DPad.LEFT).onTrue(new InstantCommand(() -> m_intake.stop(), m_intake));
+      test.get(DPad.DOWN).onTrue(new InstantCommand(() -> m_intake.intake(IntakeConstants.kIntakeSpeed), m_intake));
+      test.get(DPad.UP).onTrue(new InstantCommand(() -> m_intake.intake(IntakeConstants.kOuttakeSpeed),m_intake));
+      test.get(DPad.LEFT).onTrue(new InstantCommand(() -> m_intake.stop(), m_intake));
+    
+    // elevator controls
+    if (m_elevator != null) {
+      //test.get(DPad.DOWN).onTrue(new InstantCommand(() -> m_elevator.setMotorPower(-0.1), m_elevator));
+      //test.get(DPad.UP).onTrue(new InstantCommand(() -> m_elevator.setMotorPower(0.1),m_elevator));
+      
+      test.get(DPad.LEFT).onTrue(new InstantCommand(() -> m_elevator.setMotorPower(0), m_elevator));
+      test.get(DPad.DOWN).onTrue(new CalibrateElevator(m_elevator));
+      
+
+      //test.get(Button.A).onTrue(new MoveToHeight(m_elevator, ElevatorConstants.kHeight));
+      // //move to mid node height
+      //test.get(Button.B).onTrue(new MoveToHeight(m_elevator, ElevatorConstants.kHeightMiddleNode));
+      // //move to top node height
+      //test.get(Button.Y).onTrue(new MoveToHeight(m_elevator, ElevatorConstants.kHeightTopNode));   
+      
+
+    }
```

Comment:
I would suggest useing 1 space between the commands instead of 2 but that is more personal prefence. but the 2 lines at the end needs to go.